CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [#2514](https://github.com/ruby-grape/grape/pull/2514): Add rails 8.0 to CI - [@ericproulx](https://github.com/ericproulx).
 * [#2516](https://github.com/ruby-grape/grape/pull/2516): Dynamic registration for parsers, formatters, versioners - [@ericproulx](https://github.com/ericproulx).
 * [#2518](https://github.com/ruby-grape/grape/pull/2518): Add ruby 3.4 to CI - [@ericproulx](https://github.com/ericproulx).
+* [#2529](https://github.com/ruby-grape/grape/pull/2529): Change Grape::API's `@setup` var to an array (from a set) - [@Haerezis](https://github.com/Haerezis).
```

Comment:
This is actually a bug fix, should go below under fixes with something like "Fix `route_setting` used with multiple `mount`".

Reply:
I changed the changelog and put the change in fixes section with a title I thought fit better.